transcom/mymove
# MB-16956: Add fields for sit service items

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16956)

## Summary

This updates the db to add a field to store when the customer requested delivery and when the customer contacted the prime.

These fields are then propagated throughout the prime APIs.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Have a move that is available to the prime. Add and approve a DDDSIT service item.
2. As the prime you can use the [update API](https://transcom.github.io/mymove-docs/api/prime/v1#tag/mtoServiceItem/operation/updateMTOServiceItem) to update these two new fields. You should see the new fields in the response.
3. You should also see these fields when you use the get API for either [v1](https://transcom.github.io/mymove-docs/api/prime/v1#tag/moveTaskOrder/operation/getMoveTaskOrder) or [v2](https://transcom.github.io/mymove-docs/api/prime/v2/#tag/moveTaskOrder/operation/getMoveTaskOrder).

### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots

![image](https://github.com/transcom/mymove/assets/84742904/a35b39c2-53af-4695-b08f-146119c810ce)
